conf/quincy/cephadm/sanity-cephadm.yaml

```diff
@@ -0,0 +1,68 @@
+globals:
```

Comment:
Could you add some documentation about the cluster layout?

IMO, could we avoid adding folder structure here and opt for using an appropriate cluster naming. For example
```
9-node-all-roles.yaml
```

Reply:
done